MonoGame.Framework/Graphics/GraphicsDevice.DirectX.cs

```diff
@@ -646,6 +651,40 @@ protected virtual void CreateDeviceResources()
             // Get Direct3D 11.1 context
             _d3dContext = _d3dDevice.ImmediateContext.QueryInterface<SharpDX.Direct3D11.DeviceContext>();
         }
+        
+        internal void PlatformSetMultiSamplingToMaximum(PresentationParameters presentationParameters, out int quality)
+        {
+            var format = presentationParameters.BackBufferFormat == SurfaceFormat.Color ?
+                               SharpDX.DXGI.Format.B8G8R8A8_UNorm :
+                               SharpDXHelper.ToFormat(presentationParameters.BackBufferFormat);
+
+            // Check that the multisample count specified by the game is valid.
+            var msc = PresentationParameters.MultiSampleCount;
+            if (!((msc != 0) && ((msc & (msc - 1)) == 0)))
+            {
+                throw new ApplicationException(
+                    "The specified multisample count is not a power of 2 (it must be " +
+                    "a value such as 0, 1, 2, 4, 8, 16, 32, etc.).");
+            }
+
+            // Find the maximum supported level starting with the game's requested multisampling level
+            // and halving each time until reaching 0 (meaning no multisample support).
+            var qualityLevels = 0;
+            var maxLevel = msc;
```

Comment:
By setting the maximum level to the game's desired level, it resolves @KonajuGames's issue as well.